### PR TITLE
easier way to install our conda environment; closes #179

### LINKED ISSUE #179: Easier way to install our Conda env
@rkdarst found that you can do this:
```
conda env create -f https://raw.githubusercontent.com/coderefinery/software/main/environment.yml 
```

### PR DESCRIPTION
**Pull request by bast**

### CODE REVIEW

File: content/conda-environment.md [hunk 1]
```diff
@@ -2,20 +2,14 @@
...
-   from [CodeRefinery's software
-   repository](https://github.com/coderefinery/software/blob/main/environment.yml)
-   or make an `environment.yml` with the same contents inside the newly made
-   folder.
-3. In Bash, navigate to the folder where you put the `environment.yml`
-   (the file names the environment `coderefinery` automatically).
-4. If you have not, activate `conda` in Miniconda first using `conda activate` and if this does not
+1. Open your terminal shell (e.g. Bash)
+2. If you have not, activate `conda` in Miniconda first using `conda activate` and if this does not
    work, please first follow {ref}`setting-conda-path`.
```
Comment (rkdarst) [inline content/conda-environment.md:7]:
```suggestion
   work, please first follow {ref}`setting-conda-path`.  You probably
   need to restart your shell terminal.
```

File: content/conda-environment.md [hunk 2]
```diff
@@ -2,20 +2,14 @@
...
-5. Run the following command: `conda env create -f environment.yml`
-6. Make sure that you have "coderefinery" folder in Miniconda3/envs/. If you
+3. Run the following command:
+   ```
+   conda env create -f https://raw.githubusercontent.com/coderefinery/software/main/environment.yml
+   ```
+4. Make sure that you have "coderefinery" folder in Miniconda3/envs/. If you
    use Anaconda3, then it should be in Anaconda3/envs/.
```
Comment (rkdarst) [inline content/conda-environment.md:13]: Is it better to `conda env list` ?  Then we don't have to expect people to know where these directories are.

**Review by rkdarst — APPROVED** [review]
looks good, one suggestion/two comments

File: content/conda-environment.md (hunk 2 above)
Comment (bast) [reply content/conda-environment.md:14]: I agree. Adjusting in a follow-up commit.

**Pull request merged**